h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -860,6 +864,32 @@ public KLimeModelPrediction predictKLime(RowData data) throws PredictException {
     return p;
   }
 
+  /**
+   *  See {@link #varimp(int)}
+   *  return descending sorted by relative importance array of all variables in the model
+   */
+  public KeyValue[] varimp() {
+    return varimp(-1);
+  }
+
+  /**
+   * See {@link VariableImportances#topN(int)}
+   */
+  public KeyValue[] varimp(int n) {
+    if (m instanceof MojoModel) {
+      ModelAttributes attributes = ((MojoModel) m)._modelAttributes;
+      if (attributes == null) {
+        throw new IllegalStateException("Model attributes are not available. Did you load metadata from model? MojoModel.load(\"model\", true)");
+      } else if (attributes instanceof VariableImportancesHolder) {
+        return ((VariableImportancesHolder) attributes).getVariableImportances().topN(n);
+      } else {
+        throw new IllegalStateException("Model does not support variable importance");
+      }
+    } else {
+      throw new IllegalStateException("Model does not support variable importance");
```

Comment:
you can move the `throw new IllegalStateException("Model does not support variable importance")` outside of the if-else blocks and leave it as the last statement of the method, thus avoiding repetition

Reply:
Thanks, I will wait for jenkyns and merge it.

Reply:
:+1